src/scripts/start-marklogic.sh

```diff
@@ -503,7 +503,7 @@ do
         sudo touch /var/opt/MarkLogic/ready
         info "Cluster config complete, marking this container as ready."
         break
-    elif [[ "${MARKLOGIC_INIT}" == "false" ]] && [[ "${error_message}" =~ "SEC-DEFAULTUSERDNE" ]]; then
+    elif [[ "${MARKLOGIC_INIT}" != "true" ]] && [[ "${error_message}" =~ "SEC-DEFAULTUSERDNE" ]]; then
```

Comment:
This definitely fix part of the scenario. I was wondering can we remove the check for [[ "${error_message}" =~ "SEC-DEFAULTUSERDNE" ]]; Just mark it as ready when MARKLOGIC_INIT is set to false. Because For Kubernetes 2.0.0, the MARKLOGIC_INIT is always set to false, and there no error message because MarkLogic will be initialized by Kubernetes directly

Reply:
Removing the check for [[ "${error_message}" =~ "SEC-DEFAULTUSERDNE" ]] should be implemented as part of Kubernetes 2.0.0 change for proper tracking of change and reason.

Reply:
This is a Docker change, so it could not be tracked as Kubernetes 2.0.0. I agree to create another story to track the Kubernetes 2.0.0 change. One thing we need to know is should Docker 2.0.0 support Kubernetes 2.0.0.